tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -408,11 +409,16 @@ task CollectMultipleMetrics {
   Int max_heap = memory_mb - 500
 
   command <<<
+    #plots will not be produced if there are no reads
+    touch ~{output_bam_prefix}.insert_size_histogram.pdf
+    touch ~{output_bam_prefix}.insert_size_metrics
+    touch ~{output_bam_prefix}.base_distribution_by_cycle.pdf
+    touch ~{output_bam_prefix}.quality_by_cycle.pdf
+    touch ~{output_bam_prefix}.quality_distribution.pdf
+
     java -Xms~{java_memory_size}m -Xmx~{max_heap}m -jar /usr/picard/picard.jar CollectMultipleMetrics \
       INPUT=~{input_bam} \
       OUTPUT=~{output_bam_prefix} \
-      PROGRAM=CollectInsertSizeMetrics \
```

Comment:
These do not be specified explicitly, because by default CollectMultipleMetrics runs CollectAlignmentSummaryMetrics, CollectBaseDistributionByCycle, CollectInsertSizeMetrics, MeanQualityByCycle, and QualityScoreDistribution.  Adding them explicitly (without also `PROGRAM=null` to clear the defaults) has no effect